src/main/java/com/jmb/springfactory/service/productionorder/ProductionOrderServiceImpl.java

```diff
@@ -66,26 +77,65 @@ public ProductionOrderDTO findOneByIdInSchedule(Integer idSchedule, Integer idOr
     public ProductionOrderDTO save(final ProductionOrderDTO order, Integer idSchedule)
             throws ServiceLayerException, NotFoundException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)